### PR TITLE
Add elementUI, sass and sass-loader@7.3.1

### PR DESCRIPTION
**Pull request by WuChuSheng1**
* Add elementUI, sass and sass-loader@7.3.1
* Initialize page structure
  * Add Database.vue and Structure.vue
  * delete Laws.vue
* Add Header Component
  * Add NavMenu from ElementUI

### CODE REVIEW

**Comment by wu-sheng** [comment]
The license should be resolved every time a new dependency is being added.

**Comment by wu-sheng** [comment]
https://github.com/apache/skywalking-eyes eyes tool has been enhanced to check license automatically with certain ordered license lists.

Zhenxu is doing the same for the main repo, https://github.com/apache/skywalking/pull/9220.

Please make sure dependencies would be checked in the CI process.

**Comment by hanahmily** [comment]
@WuChuSheng1 Please refer to https://github.com/apache/skywalking-banyandb/blob/main/CONTRIBUTING.md#test-your-changes-before-pushing before pushing a commit.

File: ui/embed.go [hunk 1]
```diff
@@ -20,5 +20,5 @@ package ui
 import "embed"
 
 // DistContent contains ui distributed pages
-//go:embed dist
+// go:embed dist
```
Comment (lujiajing1126) [inline ui/embed.go:23]: Why adding a space here?

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

**Comment by WuChuSheng1** [comment]
> Where are notice.png and person.png from? Which licenses are they using?

They come from Alibaba Icon.

File: ui/embed.go (hunk 1 above)
Comment (WuChuSheng1) [reply ui/embed.go:23]: Because my vscode is wrong about this place, If this place is not well done, I will revise it back.

**Comment by Superskyyy** [comment]
> > Where are notice.png and person.png from? Which licenses are they using?
> 
> They come from Alibaba Icon.

Are you saying these icons are from Alibaba's IconFont? E.g. here https://www.iconfont.cn/search/index?searchType=icon&q=notice&page=1&tag=

If so I believe we are gonna have a copyright problem, it's not licensed for commercial use.

**Comment by WuChuSheng1** [comment]
> > > Where are notice.png and person.png from? Which licenses are they using?
> > 
> > 
> > They come from Alibaba Icon.
> 
> Are you saying these icons are from Alibaba's IconFont? E.g. here https://www.iconfont.cn/search/index?searchType=icon&q=notice&page=1&tag=
> 
> If so I believe we are gonna have a copyright problem, icons on this site are not licensed for commercial use and the copyright belongs to the icon owner and the platform.

Yes, they are from alibaba's iconfont.However, Zhenxu suggested that I use open source icons such as awesomefont, so I will use this type of icon in the future.

File: ui/embed.go (hunk 1 above)
Comment (lujiajing1126) [reply ui/embed.go:23]: Pls revert this line

**Comment by Superskyyy** [comment]
> > > > Where are notice.png and person.png from? Which licenses are they using?
> > > 
> > > 
> > > They come from Alibaba Icon.
> > 
> > 
> > Are you saying these icons are from Alibaba's IconFont? E.g. here [iconfont.cn/search/index?searchType=icon&q=notice&page=1&tag=](https://www.iconfont.cn/search/index?searchType=icon&q=notice&page=1&tag=)
> > If so I believe we are gonna have a copyright problem, icons on this site are not licensed for commercial use and the copyright belongs to the icon owner and the platform.
> 
> Yes, they are from alibaba's iconfont.However, Zhenxu suggested that I use open source icons such as awesomefont, so I will use this type of icon in the future.

Yeah, that's a good idea, probably we should replace these icons here in this PR. I like font awesome, but maybe also take a look at [material-design-icons](https://github.com/google/material-design-icons) they look neat.

**Comment by WuChuSheng1** [comment]
> Yeah, that's a good idea, probably we should replace these icons here in this PR. I like font awesome, but maybe also take a look at [material-design-icons](https://github.com/google/material-design-icons) they look neat.

OK, thank you for your sincere advice！

File: ui/embed.go (hunk 1 above)
Comment (WuChuSheng1) [reply ui/embed.go]: ok,I will.

**Comment by wu-sheng** [comment]
We need our licenses of any files copied from others clean. So choose clearly, and update license file accoedingly. 
In this case(file copy), both licenses for source codes and binary(dist) should be updated both.

**Review by hanahmily — APPROVED** [review]
LGTM

**Review by lujiajing1126 — DISMISSED** [review]

**Comment by Superskyyy** [comment]
It's probably not needed to run a 40+ minute long e2e testing of BanyanDB itself if changes are only targeting UI side right? Maybe you could make two [action rules](https://github.com/apache/skywalking/blob/46ae1cfe1ea137577a6d191f4bda39ea82feb2ac/.github/workflows/skywalking.yaml#L69) like this to only run UI-related jobs when only UI is changed, same for db part.  @WuChuSheng1

**Comment by WuChuSheng1** [comment]
> It's probably not needed to run a 40+ minute long e2e testing of BanyanDB itself if changes are only targeting UI side right? Maybe you could make two [action rules](https://github.com/apache/skywalking/blob/46ae1cfe1ea137577a6d191f4bda39ea82feb2ac/.github/workflows/skywalking.yaml#L69) like this to only run UI-related jobs when only UI is changed, same for db part. @WuChuSheng1

I agree with you very much. I will study and adopt this method in the future.

**Comment by wu-sheng** [comment]
I think this PR was approved? It is a better practice we keep a PR smaller.

**Comment by WuChuSheng1** [comment]
> I think this PR was approved? It is a better practice we keep a PR smaller.

Yes, you are right. But I don't know that after the PR is approved, the submitted code will reject the old comments of lujiajing 1126 and hanahmily. I'm very sorry.

**Comment by wu-sheng** [comment]
Could you revert the commits back? And make CI passed?

It is not rejected, you still could submit codes if necessary, such as bugs or code polish. I just mentioned this, in case codes keep being submitted, and LOC of this PR keeps growing.

**Comment by WuChuSheng1** [comment]
> Could you revert the commits back? And make CI passed?

of course.

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
I forwarded @hanahmily and @lujiajing1126 's approvals.

**Pull request merged**